admin_manual/configuration_user/profile_configuration.rst

```diff
@@ -0,0 +1,28 @@
+=====================
```

Comment:
I suppose this needs to be somehow included into the doctree?
(github actions warning)

Reply:
Yeah in https://github.com/nextcloud/documentation/blob/a6197f77d3c41e63104dd5a589add965f53dab9c/admin_manual/configuration_user/index.rst